src/commands/appSettings/AzureWebJobsStoragePromptStep.ts

```diff
@@ -6,52 +6,64 @@
 import { IAzureWebJobsStorageWizardContext } from './IAzureWebJobsStorageWizardContext';
+import { IConnectionPromptOptions } from './IConnectionPromptOptions';
 
 export class AzureWebJobsStoragePromptStep<T extends IAzureWebJobsStorageWizardContext> extends AzureWizardPromptStep<T> {
-    private readonly _suppressSkipForNow?: boolean;
-
-    public constructor(suppressSkipForNow?: boolean) {
+    public constructor(private readonly _options?: IConnectionPromptOptions) {
         super();
-        this._suppressSkipForNow = suppressSkipForNow;
     }
 
     public async prompt(context: T): Promise<void> {
-        const selectAccount: MessageItem = { title: localize('selectAzureAccount', 'Select storage account') };
-        const useEmulator: MessageItem = { title: localize('userEmulator', 'Use local emulator') };
-        const skipForNow: MessageItem = { title: localize('skipForNow', 'Skip for now') };
+        const connectStorageButton: MessageItem = { title: localize('connectStorageAccount', 'Connect Storage Account') };
+        const useEmulatorButton: MessageItem = { title: useEmulator };
+        const skipForNowButton: MessageItem = { title: skipForNow };
 
-        const message: string = localize('selectAzureWebJobsStorage', 'In order to debug, you must select a storage account for internal use by the Azure Functions runtime.');
+        const message: string = localize('connectAzureWebJobsStorage', 'In order to proceed, you must connect a storage account for internal use by the Azure Functions runtime.');
 
-        const buttons: MessageItem[] = [selectAccount];
-        if (process.platform === 'win32') {
-            // Only show on Windows until Azurite is officially supported: https://github.com/Azure/azure-functions-core-tools/issues/1247
-            buttons.push(useEmulator);
-        }
-        if (!this._suppressSkipForNow) {
-            buttons.push(skipForNow);
+        const buttons: MessageItem[] = [connectStorageButton, useEmulatorButton];
+
+        if (!this._options?.suppressSkipForNow) {
+            buttons.push(skipForNowButton);
         }
 
         const result: MessageItem = await context.ui.showWarningMessage(message, { modal: true }, ...buttons);
-        if (result === selectAccount) {
-            context.azureWebJobsStorageType = 'azure';
-        } else if (result === useEmulator) {
-            context.azureWebJobsStorageType = 'emulator';
+        if (result === connectStorageButton) {
+            context.azureWebJobsStorageType = ConnectionType.Azure;
+        } else if (result === useEmulatorButton) {
+            context.azureWebJobsStorageType = ConnectionType.NonAzure;
+        } else {
+            context.azureWebJobsStorageType = ConnectionType.None;
         }
 
-        context.telemetry.properties.azureWebJobsStorageType = context.azureWebJobsStorageType || 'skipForNow';
+        context.telemetry.properties.azureWebJobsStorageType = context.azureWebJobsStorageType;
```

Comment:
Will the telemetry property for `skipForNow` be `None` after this change? 

This is a potential data point for us to see if people are dropping in the funnel here, so it'd be good if we could keep this consistent with how it used to be.